bridge/opentracing/bridge.go

```diff
@@ -91,10 +89,10 @@ func (s *bridgeSpan) Finish() {
 }
 
 func (s *bridgeSpan) FinishWithOptions(opts ot.FinishOptions) {
-	var otelOpts []oteltrace.EndOption
+	var otelOpts []otel.EndOption
```

Comment:
One of the problems that we get by merging all this packages is the mess with functional options. With your changes we someone is starting a span, finishing or doing another operation that accepts options how can one know what options are available to them?

Sure you can see the return type of each `WithOption` but it much better if we can decrease these options based on the operation. E.g., creating a span:
![image](https://user-images.githubusercontent.com/7898464/68443857-eb7f8c80-01b3-11ea-99c8-c791f7beed62.png)


I don't know how this could be solved, would you consider creating `traceopts`, `metricsopts` and etc?